### ISSUE TITLE
install.sh-script is using bash specifics while using sh in the hashbang

### ISSUE THREAD

**Issue by jerri** [kkoomen/vim-doge#160]
Updating vim-doge today on my computer led to the following errors

```
> ./scripts/install.sh
./scripts/install.sh: 17: [[: not found
/usr/bin/curl
./scripts/install.sh: 24: [[: not found
./scripts/install.sh: 31: [[: not found
./scripts/install.sh: 33: [[: not found
vim-doge does not support Windows yet
```

I am on Ubuntu here. I think the problem is related to `sh` being linked to `dash` on this computer, whereas the script used `bash`-extensions that `dash` does not support. I would suggest to either change the first line of the script to `#!/usr/bin/env bash`, or not use `[[` in the script.

**To Reproduce**

Executing the script on a linux system that differentiates between sh and bash will lead to an error.

**Comment by kkoomen** [comment]
That's a problem that happened due to the merge from yesterday. We decided to use `sh` as it's installed on almost every system. We can't guarantee bash is installed in all systems. I'll make it compatible with dash.

**Comment by kkoomen** [comment]
This bug has been fixed and released in [v3.5.3](https://github.com/kkoomen/vim-doge/tree/v3.5.3).

Feel free to submit any new issues if you experience any unwanted behavior in the future. Thanks for your contribution.

**Comment by iFork** [comment]
@kkoomen - fix does not affect line 33 `elif [[ $OS == 'Linux' ]]; then` which still gives error. Please fix that line, too.

**Comment by kkoomen** [comment]
This bug has been fixed and released in [v3.5.4](https://github.com/kkoomen/vim-doge/tree/v3.5.4).

Feel free to submit any new issues if you experience any unwanted behavior in the future. Thanks for your contribution.
